app/validators/gov_uk_url_format_validator.rb

```diff
@@ -11,4 +11,15 @@ def validate_each(record, attribute, value)
   def failure_message
     options[:message] || "must be in the form of #{Whitehall.public_protocol}://#{Whitehall.public_host}/example"
   end
+
+  def matches_gov_uk?(value)
+    %r{\A#{Whitehall.public_protocol}://#{Whitehall.public_host}/}.match?(value)
+  end
+
+  def matches_allow_list?(value)
+    uri = URI.parse(value)
+    uri.host&.end_with?(".gov.uk", ".judiciary.uk", "etl.beis.gov.uk")
```

Comment:
This is currently a little confusing, as I'd expect `.gov.uk` to only be checked in `matches_gov_uk?`, and `matches_allow_list?` to only contain the special domains.

```suggestion
    uri.host&.end_with?(".judiciary.uk", "etl.beis.gov.uk")
```

Reply:
Good spot!  I just copied this from the `publishing-api` PR 🤦